### ISSUE TITLE
Notebook: Alt-Enter should move keyboard focus to newly created cell

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode#94096]
1) create a new gh issues nb
2) place keyboard focus in cell 1
3) Alt+Enter
=> keyboard focus should be in cell 2

**Comment by rebornix** [comment]
Already fixed.
